src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java

```diff
@@ -231,6 +236,10 @@ public String getNetworking() {
     public boolean hasPortMappings() {
       return portMappings != null && !portMappings.isEmpty();
     }
+
+    public Boolean getDockerPrivilegedMode() {
+      return dockerPrivilegedMode != null ? dockerPrivilegedMode : false;
```

Comment:
I Thougth this should be the default setting for docker containers. Nevertheless, a default setting will be set on config save, I'll change that